pkg/services/ngalert/accesscontrol.go

```diff
@@ -55,6 +55,9 @@ var (
 					Action: accesscontrol.ActionAlertingRuleExternalWrite,
 					Scope:  datasources.ScopeAll,
 				},
+				{
+					Action: accesscontrol.ActionAlertingReceiversList,
+				},
```

Comment:
Let's add to reader, and writer will get it for free. Person that opens Modify Export should probably be able to pick contact point